docs/metal_project_bgp-config.md

```diff
@@ -0,0 +1,44 @@
+## metal project bgp-config
+
+Gets BGP Config for a project.
+
+### Synopsis
+
+Gets BGP Config for a project.
+
+```
+metal project bgp-config --id <project_UUID> [flags]
+```
+
+### Examples
+
+```
+  # Get BGP config for project 50693ba9-e4e4-4d8a-9eb2-4840b11e9375:
+  metal project bgp-config --id 50693ba9-e4e4-4d8a-9eb2-4840b11e9375
+```
+
+### Options
+
+```
+  -h, --help        help for bgp-config
+  -i, --id string   Project ID (METAL_PROJECT_ID)
```

Comment:
`METAL_PROJECT_ID` is only used when the argument is `--project-id`.

There are a few places in `metal project` where this environment variable is incorrectly suggested (including here).

Reply:
hmmm... do you know which those are, I've been trying to spot check it as I go along. Even better if we could make that behavior consistent.

Reply:
Oh, do you mean there is some actual, meaningful distinction between when the flag is `--id` vs `--project-id`? Because that's....not great?

Reply:
`--id` tends to be relative to the resource you are working with in each command.

`--project-id` is a standard, almost global, argument because it is so frequently needed.

Because `--project-id` inherits from the `metal init` config, to prevent the default project from being applied on all project commands (like `delete`) we avoided `--project-id` in the `metal project` set of commands. In favor of that, we use `--id` (the action relative parameter).

In any case, `METAL_PROJECT_ID` only maps to `--project-id` arguments, not `--id`.

When we start talking about `metal project bgp*` the dominant prevalence of Project and BGP is called into question when we think about `--id`.

Reply:
hmmmmm......ok. That makes more sense than what I was thinking. And now I see the conflict here. So is if it is ok to use `--project-id` for project ID and `--id` for the BGP id? 

In this case, probably yes? because if --project-id is pulled in from the environment it won't supercede any other behavior. I think. Like there is nothing to be expected if you just send in `metal bgp get`. It would be ok for `metal bgp get` to just return the same thing as `metal bgp get --project-id <project_id>.

So I think I would use `--project-id` and `--id` here. Unless I am missing something.

Reply:
That makes sense.